instance/default.cfg

```diff
@@ -2,3 +2,7 @@ SQLALCHEMY_DATABASE_URI = 'sqlite:///test.db'
 SQLALCHEMY_TRACK_MODIFICATIONS = False
 JENKINS_URL = 'https://build.gluster.org'
 JENKINS_MAX = 1000
+DEBUG = True
```

Comment:
Stop adding secret keys and client_ids into `default.cfg`. You need to put this into your own `application.cfg`. That's specifically ignored by git so you can put app specific configuration.